app/client/ui/ToggleEnterprisePage.ts

```diff
@@ -0,0 +1,88 @@
+import {makeT} from 'app/client/lib/localization';
+import {Computed, Disposable, dom, makeTestId} from "grainjs";
+import {AppModel} from "app/client/models/AppModel";
+import {loadingSpinner} from 'app/client/ui2018/loaders';
+import {commonUrls} from "app/common/gristUrls";
+import {cssLink} from 'app/client/ui2018/links';
+import {ToggleEnterpriseModel} from 'app/client/models/ToggleEnterpriseModel';
+import {
+  cssOptInButton,
+  cssOptInOutMessage,
+  cssOptOutButton,
+  cssParagraph,
+  cssSection,
+  cssSpinnerBox
+} from 'app/client/ui/AdminTogglesCss';
+
+
+const t = makeT('ToggleEnterprsiePage');
+const testId = makeTestId('test-toggle-enterprise-page-');
+
+export class ToggleEnterprisePage extends Disposable {
```

Comment:
This is not a page, is it? Just a container for those two paragraphs? Just a suggestion to change the name to ToggleEnterpriseWidgets/Helpers or something like that.